client/src/components/TextArea/Text.js

```diff
@@ -0,0 +1,12 @@
+import React from 'react'
+import './Text.css'
+
+
+const Text = (props) => <div>
```

Comment:
Can you call the component `TextArea`? Also the name of the file should be `TextArea.js`